data/links.yaml

```diff
@@ -1101,6 +1101,11 @@ links:
     url: "https://myaccount.ft.com/details/core/view"
     submenu:
 
+  - &contact_preferences
+    label: "Contact Preferences"
+    url: "/myft/alerts"
```

Comment:
I think this url may be incorrect, it doesn't read like it will take someone to contact preferences